commands/suggest_stacks.go

```diff
@@ -1,16 +1,17 @@
 package commands
 
 import (
-	"github.com/buildpack/pack/logging"
 	"github.com/spf13/cobra"
+
+	"github.com/buildpack/pack/logging"
 )
 
-func SuggestStacks(logger *logging.Logger) *cobra.Command {
+func SuggestStacks(logger logging.Logger) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "suggest-stacks",
 		Short: "Display list of recommended stacks",
 		Args:  cobra.NoArgs,
-		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
+		RunE: logError(nil, func(cmd *cobra.Command, args []string) error {
```

Comment:
I don't think we should be passing nil here. `logError` may need to handle/log a syntax error